spec/predicates/vuln.md

```diff
@@ -64,41 +64,39 @@ The `predicate` contains a JSON-encoded data with the following fields:
 
 > > > The version of the Vulnerability DB.
 
-**scanner.db.lastUpdate, required** string (timestamp)
+**scanner.db.lastUpdate, optional** string (timestamp)
```

Comment:
I won't change this setting. It is important to reflect the update of the database used for the scanner.

Reply:
i find that this information is generally not very available. There usually is a last update on an entry of a vuln DB, but the lastUpdate of a vuln database i don't believe i've been able to populate (i'm thinking about OSV and MITRE db)

Reply:
This information is relevant for many scanners such as Grype, depending on the date of last update the report could be discarded.